.github/workflows/integration_test.yml

```diff
@@ -106,8 +106,8 @@ jobs:
           make chaosctl
       - name: Install localstack && aws client
         run: |
-          helm repo add localstack-repo http://helm.localstack.cloud
-          helm upgrade --install localstack localstack-repo/localstack --version 0.1.2
+          helm repo add localstack-repo http://helm.localstack.cloud && helm repo update
```

Comment:
Should we need to run `helm repo update`? The `helm repo add` command should already get the latest version. 🤔

Reply:
Yes, you're right, we do not need `helm repo update` for environments in github actions.